pkg/networkservice/common/clientinfo/client_test.go

```diff
@@ -0,0 +1,115 @@
+package clientinfo
+
+import (
+	"context"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/networkservicemesh/api/pkg/api/networkservice"
+	"github.com/networkservicemesh/sdk/pkg/networkservice/core/next"
+)
+
+type clientInfoTestData struct {
```

Comment:
FYI... you could use anonymous structures for test samples

Reply:
Ok, i will use them